java/bench/pom.xml

```diff
@@ -17,10 +17,10 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache</groupId>
-    <artifactId>apache</artifactId>
-    <version>18</version>
-    <relativePath></relativePath>
+    <groupId>org.apache.orc</groupId>
+    <artifactId>orc</artifactId>
+    <version>1.7.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
```

Comment:
This seems to break uber jar building in `spark` submodule.
```
$ cd java/bench

$ mvn clean package -pl spark --am

$ ls -alh spark/target/*.jar      
-rw-rw-r-- 1 dongjoon dongjoon  62K Mar 23 05:28 spark/target/orc-benchmarks-spark-1.7.0-SNAPSHOT.jar
-rw-rw-r-- 1 dongjoon dongjoon 808K Mar 23 05:28 spark/target/orc-benchmarks-spark-1.7.0-SNAPSHOT-nohive.jar
```

The correct uber jar should be 129MB.
```
$ ls -alh spark/target/*.jar
-rw-rw-r-- 1 dongjoon dongjoon 129M Mar 23 05:32 spark/target/orc-benchmarks-spark-1.7.0-SNAPSHOT.jar
-rw-rw-r-- 1 dongjoon dongjoon  63K Mar 23 05:32 spark/target/original-orc-benchmarks-spark-1.7.0-SNAPSHOT.jar
```

Reply:
I'll make a PR to fix this.